src/AdvisoryConsumer.cs

```diff
@@ -66,7 +67,7 @@ internal void Dispose()
             }
         }
 
-        public void Dispatch(MessageDispatch messageDispatch)
+        public Task Dispatch_Async(MessageDispatch messageDispatch)
```

Comment:
I don't like this dash here. Why not call this method just `DispatchAsync`.

Reply:
i recall there was some naming collision